content/zh/2022-03-14-the-apache-community/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: The Apache Community
```

Comment:
```suggestion
title: 我眼中的Apache Community
```

Reply:
How is this? We just need to be sure this blog is your personal opinion.

Reply:
> We just need to be sure this blog is your personal opinion.

Yep. I'd prefer:

```suggestion
title: 我眼中的 The Apache Way
```